### PR TITLE
Dummy comment added.

### PR DESCRIPTION
**Pull request by aryanagarkar**
Please review.

### CODE REVIEW

File: src/main/java/assignments/personinheritence/HumanBeing.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package assignments.personinheritence;
 
+// Dummy comment
```
Comment (nagarkar) [inline src/main/java/assignments/personinheritence/HumanBeing.java]: Add a better comment.

File: src/main/java/assignments/personinheritence/HumanBeing.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package assignments.personinheritence;
 
+// Dummy comment
 public interface HumanBeing {
 
     public abstract String introduceYourself();
```
Comment (nagarkar) [inline src/main/java/assignments/personinheritence/HumanBeing.java]: Is there a better name for this interface?

File: src/main/java/assignments/personinheritence/HumanBeing.java (hunk 1 above)
Comment (aryanagarkar) [reply src/main/java/assignments/personinheritence/HumanBeing.java]: Added a better comment

File: src/main/java/assignments/personinheritence/HumanBeing.java (hunk 2 above)
Comment (aryanagarkar) [reply src/main/java/assignments/personinheritence/HumanBeing.java]: I think the interface name makes sense.

**Comment by nagarkar** [comment]
Thanks for doing this work.

**Comment by nagarkar** [comment]
Thanks for doign this work.

**Pull request merged**